__tests__/Unit/Components/Standup/StandupContainer.test.tsx

```diff
@@ -0,0 +1,237 @@
+import { store } from '@/app/store';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { setupServer } from 'msw/node';
+import handlers from '../../../../__mocks__/handlers';
+import {
+    ERROR_MESSAGE,
+    STANDUP_SUBMISSION_SUCCESS,
+} from '@/constants/constants';
+import { failedPostStandup } from '../../../../__mocks__/handlers/standup.handler';
+import { ToastContainer } from 'react-toastify';
+import * as SaveProgressHook from '@/app/services/progressesApi';
+import moment from 'moment';
+
+const server = setupServer(...handlers);
+
+describe('StandupContainer', () => {
+    beforeAll(() => {
+        server.listen();
+    });
+    afterEach(() => {
+        server.resetHandlers();
+    });
+    afterAll(() => {
+        server.close();
+    });
+    const yesterdayDate = moment().subtract(1, 'days').format('MMMM DD, YYYY');
+
+    test('should render  inputField ', function () {
+        renderWithRouter(
+            <Provider store={store()}>
+                <StandUpContainer />
+            </Provider>
+        );
+
+        const completedInputField = screen.getByRole('textbox', {
+            name: yesterdayDate,
+        }) as HTMLInputElement;
+        const todayInputField = screen.getByRole('textbox', {
+            name: 'Today',
+        }) as HTMLInputElement;
+        const blockerInputField = screen.getByRole('textbox', {
+            name: 'Blockers',
+        }) as HTMLInputElement;
+
+        expect(completedInputField).toBeInTheDocument();
+        expect(completedInputField).toHaveValue('');
+        expect(todayInputField).toBeInTheDocument();
+        expect(todayInputField).toHaveValue('');
+        // expect(screen.queryByTestId('blockerInputField')).toBeInTheDocument();
+        expect(blockerInputField).toHaveValue('');
+
+        fireEvent.change(completedInputField, {
+            target: { value: 'Working on a backend Go project' },
+        });
+        const completedInputValue = completedInputField.value;
+        expect(completedInputValue).toBe('Working on a backend Go project');
+
+        fireEvent.change(todayInputField, {
+            target: { value: 'Implement error handling for API endpoints' },
+        });
+        const todayInputValue = todayInputField.value;
+        expect(todayInputValue).toBe(
+            'Implement error handling for API endpoints'
+        );
+        fireEvent.change(blockerInputField, {
+            target: { value: 'Waiting for database access credentials' },
+        });
+        const blockerInputValue = blockerInputField.value;
+        expect(blockerInputValue).toBe(
+            'Waiting for database access credentials'
+        );
+    });
+
+    test('render if button is disabled', () => {
+        renderWithRouter(
+            <Provider store={store()}>
+                <StandUpContainer />
+            </Provider>
+        );
+        expect(screen.getByText(/Submit/i).closest('button')).toBeDisabled();
+    });
```

Comment:
after this can you try entering text into one of the fields and ensure if the button is still disabled.

Reply:
added and it's working as per requirement.

Reply:
Where did you add it?

Reply:
![image](https://github.com/Real-Dev-Squad/website-status/assets/82165483/e3d7b86b-b6cf-497a-88a3-ccc8680c019d)